packages/app/src/app/pages/Dashboard/Content/routes/Beta/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { useAppState, useActions } from 'app/overmind';
+import { Header } from 'app/pages/Dashboard/Components/Header';
+import { VariableGrid } from 'app/pages/Dashboard/Components/VariableGrid';
+import {
+  DashboardBetaRepo,
+  DashboardGridItem,
+  PageTypes,
+} from 'app/pages/Dashboard/types';
+import { SelectionProvider } from 'app/pages/Dashboard/Components/Selection';
+
+export const BetaRepositoriesPage = () => {
+  const actions = useActions();
+  const {
+    activeTeam,
+    dashboard: { sandboxes },
+  } = useAppState();
+
+  React.useEffect(() => {
+    actions.dashboard.getBetaSandboxes();
+  }, [actions.dashboard]);
+
+  const items: DashboardGridItem[] = sandboxes.BETA
+    ? [
+        { type: 'beta-new-repo' },
+        ...sandboxes.BETA.map(sandbox => ({
+          type: 'beta-repo' as DashboardBetaRepo['type'],
```

Comment:
Should work to convert the type from 'string' to `'beta-repo'`:

```suggestion
          type: 'beta-repo' as const,
```

Reply:
Good to know!